src/Backups/BackupsWorker.cpp

```diff
@@ -504,6 +504,8 @@ OperationID BackupsWorker::startMakingBackup(const ASTPtr & query, const Context
                  context_in_use,
                  mutable_context,
                  on_exception,
+                 /// Capture original context to preserve Settings/current_database in system.processes
+                 context,
```

Comment:
I come to the same idea of keeping the original context while I was reworking on https://github.com/ClickHouse/ClickHouse/pull/70027

Reply:
OK, can you then please copy my tests then?